ParabolInc/parabol
# fix(demo): card quick return to original position without hangs

# Description

Fixes/Partially Fixes #5975
When proceeding from the Group phase to the Vote phase in the retro demo, the card that was being dragged hangs and takes a while to return to its original position.

## Demo

https://user-images.githubusercontent.com/4997466/187667074-b3de2dc4-8250-4c30-b332-99bd4bb6b706.mp4

## Testing scenarios

- start the retro demo
- go to the Group phase, note that the grouping isn't finished yet 
- move forward to Vote phase, check the dragging card quick transition and return to the original position

## Final checklist
